File: src/app/classes/Color.ts

export class Color {
    red: number;
    green: number;
    blue: number;
    alpha: number;
    constructor(red: number,
                green: number,
                blue: number,
                alpha: number)
        {
            this.red = red;
            this.green = green;
            this.blue = blue;
            this.alpha = alpha;
        }
    toString(): string{
        return `rgba(${this.red}, ${this.green}, ${this.blue}, ${this.alpha})`;
    }
}

File: src/main/java/com/PaintApp/PaintAppBackend/Shapes/Color.java
package com.PaintApp.PaintAppBackend.Shapes;

public class Color {
    private int red;
    private int green;
    private int blue;
    private int alpha;

    Color(int red, int green, int blue, int alpha){
        this.red = red;
        this.green = green;
        this.blue = blue;
        this.alpha = alpha;
    }

    public String toString(){
        return "rgba(" + this.red + "," + this.green + "," + this.blue + "," + this.alpha + ")";
    }
}
